clinica/iotools/converters/adni_to_bids/adni_to_bids.py

```diff
@@ -38,15 +38,17 @@ def get_bids_subjs_info(
 
 
 class AdniToBids(Converter):
-    def get_modalities_supported(self):
+    @classmethod
```

Comment:
Is there a reason you decided to use a class method instead of a static method (or a `supported_modalities` property) here?

Reply:
This code is not specific to any class instance (`self` is unused) so it makes sense to turn into a class method.